about.html

```diff
@@ -0,0 +1,198 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Document</title>
+    <link rel="stylesheet" href="./styles.css" />
+
+    <link
+      href="https://cdn.jsdelivr.net/npm/bootstrap@5.1.3/dist/css/bootstrap.min.css"
+      rel="stylesheet"
+    />
+    <script src="https://cdn.jsdelivr.net/npm/bootstrap@5.1.3/dist/js/bootstrap.bundle.min.js"></script>
+    <script
+      src="https://kit.fontawesome.com/00fad0ddbb.js"
+      crossorigin="anonymous"
+    ></script>
+  </head>
+  <body>
+    <header>
+      <div class="social-media">
+        <div></div>
+        <div class="social-media-links">
+          <i class="fa-brands logo-icon fa-square-facebook"></i>
+          <i class="fa-brands logo-icon fa-twitter"></i>
+          <a href="">English</a>
+          <a href="">My Page</a>
+          <a href="">Logout</a>
+        </div>
+      </div>
+
+      <nav class="navbar nav-mobile disappearing-nav1 padd">
+        <div class="hamburger">
+          <div id="myHamburger" class="ham-container hamburger">
+            <div class="bar1"></div>
+            <div class="bar2"></div>
+            <div class="bar3"></div>
+          </div>
+          <div id="myDropdown" class="dropdown-content">
+            <a href="./home.html" class="nav-link">Home</a>
+            <a href="./about.html" class="nav-link">About</a>
+            <a href="#Contact" class="nav-link">Venue</a>
+          </div>
+        </div>
+      </nav>
+
+      <nav class="menu-bar">
+        <div class="topnav">
+          <div class="bx1">
+            <a href="home.html" class="logo-txt"
+              >F<i class="fa-brands fa-google"></i> MusicFest</a
+            >
+          </div>
+          <div class="bx2">
+            <a href="home.html">Home</a>
+            <a href="about.html">About</a>
+            <a href="#">Venue</a>
+          </div>
+        </div>
+      </nav>
+    </header>
+    <section class="first-sec">
```

Comment:
- [ ] Kindly check on the design of this section, the styling and background doesn't match the design given below is a comparison for better understanding:

Behance Design          |  Your design     
:-------------------------:|:-------------------------:
![image](https://user-images.githubusercontent.com/61408860/181731751-e6b34d83-b806-4bbc-be91-36db77576048.png) | ![image](https://user-images.githubusercontent.com/61408860/181731801-8db4759b-7627-4386-86b4-f114bc267f94.png)